CHANGELOG.md

```diff
@@ -3,6 +3,79 @@
+
+The APQ layer will now report cache hits and misses to Apollo Studio if telemetry is configured
+
+By [@Geal](https://github.com/Geal) in https://github.com/apollographql/router/pull/1117
+
+### Add more information to the `subgraph_request` span ([PR #1119](https://github.com/apollographql/router/pull/1119))
+
+Add a new span only for the subgraph request, with all HTTP and net information needed for the OpenTelemetry specs.
+
+By [@bnjjj](https://github.com/bnjjj) in https://github.com/apollographql/router/pull/1119
+
+## 🐛 Fixes
+
+### Compute default port in span information ([Issue #1160](https://github.com/apollographql/router/pull/1160)) 
+
+Compute default port in span information for `net.peer.port` regarding the scheme of the request URI.
+
+By [@bnjjj](https://github.com/bnjjj) in https://github.com/apollographql/router/pull/1160
+
+### Response `Content-Type` is, again, `application/json` ([Issue #636](https://github.com/apollographql/router/issues/636)) 
+
+The router was not setting a `content-type` on client responses. This fix ensures that a `content-type` of `application/json` is set when returning a GraphQL response.
+
+By [@bnjjj](https://github.com/bnjjj) in https://github.com/apollographql/router/pull/1154
+
+### Prevent memory leaks when tasks are cancelled ([PR #767](https://github.com/apollographql/router/pull/767))
+
+Cancelling a request could put the router in an unresponsive state where the deduplication layer or cache would make subgraph requests hang.
+
+By [@Geal](https://github.com/Geal) in https://github.com/apollographql/router/pull/767
+
+## 🛠 Maintenance
+
+### Use subgraphs deployed on Fly.io in CI ([PR #1090](https://github.com/apollographql/router/pull/1090))
+
+The CI needs some Node.js subgraphs for integration tests, which complicates its setup and increases the run time. By deploying, in advance, those subgraphs on Fly.io, we can simplify the CI run.
+
+By [@Geal](https://github.com/Geal) in https://github.com/apollographql/router/pull/1090
+
+### Unpin schemars version ([Issue #1074](https://github.com/apollographql/router/issues/1074))
+
+[`schemars`](https://docs.rs/schemars/latest/schemars/) v0.8.9 caused compile errors due to it validating default types.  This change has, however, been rolled back upstream and we can now depend on `schemars` v0.8.10.
+
+By [@o0Ignition0o](https://github.com/o0Ignition0o) in https://github.com/apollographql/router/pull/1135
+
+### Update Moka to fix occasional panics on AMD hardware ([Issue #1137](https://github.com/apollographql/router/issues/1137))
+
+Moka has a dependency on Quanta which had an issue with AMD hardware. This is now fixed via https://github.com/moka-rs/moka/issues/119
+
+By [@BrynCooke](https://github.com/BrynCooke) in [`6b20dc85`](https://github.com/apollographql/router/commit/6b20dc8520ca03384a4eabac932747fc3a9358d3)
+
+## 📚 Documentation
+
+## 🐛 Fixes
```

Comment:
```suggestion
```

shall we remove these sections since they re empty ?

Reply:
I added the rhai PR to documentation, since there was a documentation change in that PR. I removed the empty Fixes (which I don't think should be there anyway): 912b9852b62496653a62ab3e18b3dec47c0df5e4